test-snapshots/snapshots-input/Snapshot.Exports.purs

```diff
@@ -0,0 +1,8 @@
+module Snapshot.Exports
+  ( TypeOnly
+  , TypeAndCtor
```

Comment:
Uhm, actually I was thinking that we wanted to test that it would *not* export the constructors, as we already know that exporting them works.

Reply:
>  as we already know that exporting them works.

Right, but there are other things we can export besides constructors. And that may have implications for later things when type classes get involved.

Reply:
Granted, at this point, I think there's been a bit of feature creep as now we're hitting a lot more pieces of code that are now relevant because of ADT conversion being done in this PR.

Reply:
Right, I will update the test then. Should we keep snapshot to test that it will non export everything blindly as well?

Reply:
> Right, I will update the test then. Should we keep snapshot to test that it will non export everything blindly as well?

I'm not sure what you mean.

Reply:
Sorry, I think I misread your first comment. So you're suggesting to change the snapshot to

```haskell
module Snapshot.Exports
  ( TypeOnly
  , TypeAndCtor(..)
  )
...
```

Right?

Reply:
Yeah.